common/src/test/java/com/datatorrent/common/util/AsyncFSStorageAgentTest.java

```diff
@@ -63,7 +63,6 @@ protected void finished(Description description)
     {
       try {
         FileUtils.deleteDirectory(new File("target/" + description.getClassName()));
-        FileUtils.deleteDirectory(new File(FileUtils.getTempDirectory(), "localcheckpoint"));
```

Comment:
Will it be deleted when storage agent closes? Why it was possible to delete the directory before?